modules/apps/calendar/calendar-service/src/main/java/com/liferay/calendar/service/impl/CalendarBookingServiceImpl.java

```diff
@@ -338,11 +341,7 @@ public String getCalendarBookingsRSS(
 			calendarBookingLocalService.getChildCalendarBookings(
 				parentCalendarBookingId);
 
```

Comment:
Hey @mrk-vi,

Have you tried to use the `filterCalendarBookings(List<CalendarBooking> calendarBookings, String actionId)` method? It seems to have the same intention as your `_filterCalendarBookingsByCalendarVisibility`.

Can you please check this?

Thx

Reply:
Hi @rafaprax, 

I've follow your suggestion, but using the method `filterCalendarBookings(List<CalendarBooking> calendarBookings, String actionId)` the result is still not desiderable:

-  an exception is thrown when an user that doesn't have anymore the `VIEW` permission on an invited calendar, try to open his own event.

- also, when an attendee hasn't replied to an invitation yet (status `PENDING`), the `masterEvent` owner can't see the attendee in the list of invitations. So if the organizer update the event, he could accindentally remove the attendee whitout seeing that.